apps/console/src/features/identity-providers/components/forms/authenticators/sms-otp-authenticator-form.tsx

```diff
@@ -349,22 +422,33 @@ export const SMSOTPAuthenticatorForm: FunctionComponent<SMSOTPAuthenticatorFormP
             validate={ validateForm }
         >
 
-            <Message
-                type={ "info" }
-                content={
-                    ( <Trans>
-                        <span>
-                            { t("console:develop.features.authenticationProvider.forms.authenticatorSettings" +
-                                ".smsOTP.forTestingOnlyNotice.firstLine") }
-                        </span>
-                        <Divider hidden fitted/>
-                        <span>
-                            <Icon name="info circle" style={ { visibility: "hidden" } }/>
-                            { t("console:develop.features.authenticationProvider.forms.authenticatorSettings" +
-                                ".smsOTP.forTestingOnlyNotice.secondLine") }
-                        </span>
-                    </Trans> ) }
-                width={ 13 }
+            {/*<Message*/}
+            {/*    type={ "info" }*/}
+            {/*    content={*/}
+            {/*        ( <Trans>*/}
+            {/*            <span>*/}
+            {/*                { t("console:develop.features.authenticationProvider.forms.authenticatorSettings" +*/}
+            {/*                    ".smsOTP.forTestingOnlyNotice.firstLine") }*/}
+            {/*            </span>*/}
+            {/*            <Divider hidden fitted/>*/}
+            {/*            <span>*/}
+            {/*                <Icon name="info circle" style={ { visibility: "hidden" } }/>*/}
+            {/*                { t("console:develop.features.authenticationProvider.forms.authenticatorSettings" +*/}
+            {/*                    ".smsOTP.forTestingOnlyNotice.secondLine") }*/}
+            {/*            </span>*/}
+            {/*        </Trans> ) }*/}
+            {/*    width={ 13 }*/}
+            {/*/>*/}
+            <Checkbox
+                toggle
+                label={
+                    //TODO: label name refactoring
+                    "Enable SMS OTP"
```

Comment:
We can probably i18n this string

Reply:
Resolved in https://github.com/wso2/identity-apps/pull/3699/commits/087b1e409184a950309763eb50cbf62430fd51ce